src/routes/safe/components/AddressBook/ImportEntryModal/index.tsx

```diff
@@ -92,16 +91,19 @@ const ImportEntryModal = ({ importEntryModalHandler, isOpen, onClose }) => {
   const validateCsvData = (data) => {
     for (let index = 0; index < data.length; index++) {
       const entry = data[index]
-      if (!entry.data[0] || !entry.data[1]) {
+      if (!entry.data[0] || !entry.data[1] || !entry.data[2]) {
         return `Invalid amount of columns on row ${index + 2}`
       }
       // Verify address properties
       const address = entry.data[0].toLowerCase()
       if (!getWeb3().utils.isAddress(address)) {
```

Comment:
For added `isValidAddress` for this.
`import { isValidAddress } from 'src/utils/isValidAddress'`